### ISSUE TITLE
ChatGPT选项是否不再可用？ 如果我来自墨西哥，我如何付款？

### ISSUE THREAD

**Issue by Root-FTW** [1c7/Translate-Subtitle-File#66]
ChatGPT选项是否不再可用？ 如果我来自墨西哥，我如何付款？

**Comment by 1c7** [comment]
Yes, I remove ChatGPT from provider list. because it has some bug need to fixed.
Sorry, I can't accept payment outside China for now.
I can't accept credit card or paypal, because I didn't expect people outside China would need this too.
I don't have Stripe account to accept payment

Thanks for using my software

**Comment by 1c7** [comment]
I can't speak Spanish, I only know Chinese and English.

**Comment by Root-FTW** [comment]
> Owner

Is there any date when ChatGPT will return to this project?

**Comment by 1c7** [comment]
probably within 1-2 weeks (before April 25)

**Comment by Root-FTW** [comment]
> probably within 1-2 weeks (before April 25)

great, thank you!

**Comment by Root-FTW** [comment]
Will chatgpt no longer return as an option? :(

**Comment by 1c7** [comment]
I am busy on my company's project, building chatbot using LLM (like GPT-3.5 and GPT-4), I don't have time to put into this for now

**Comment by 1c7** [comment]
@zzjoychen 不确定具体时间，主要是我没时间弄

**Comment by 1c7** [comment]
<img width="910" height="840" alt="Image" src="https://github.com/user-attachments/assets/faa49ac8-8261-4f3e-83ba-19c0bc8b4182" />

I recently update the app (May 4, 2026)
but there are too many provider. I am updating each of them one by one.

I would update "OpenAI-Compatible Chat Completions" soon
